src/Features/Core/Portable/Snippets/SnippetPlaceholder.cs

```diff
@@ -10,42 +10,42 @@ namespace Microsoft.CodeAnalysis.Snippets;
 internal readonly struct SnippetPlaceholder
 {
     /// <summary>
-    /// The identifier in the snippet that needs to be renamed.
+    /// Editable text in the snippet.
     /// </summary>
-    public readonly string Identifier;
+    public readonly string Text;
```

Comment:
This can be an actual identifier, but it also can be any other text, e.g. in `if` snippet the placeholder is a full condition expression. So `Identifier` is a confusing name for this thing, renamed it to more generic `Text`